src/stock-operations/batch-no-selector/batch-no-selector.component.tsx

```diff
@@ -5,6 +5,7 @@ import { useStockItemBatchNos } from './batch-no-selector.resource';
 import { StockBatchDTO } from '../../core/api/types/stockItem/StockBatchDTO';
 import { useStockItemBatchInformationHook } from '../../stock-items/add-stock-item/batch-information/batch-information.resource';
 import { useTranslation } from 'react-i18next';
+import { formatForDatePicker } from '../../constants';
```

Comment:
Not sure why we have custom formateDatePicker. We need to start aligning with the community standard date picker formats